.env

```diff
@@ -0,0 +1,50 @@
+# Use by development docker setup.
+COMPOSE_PROJECT_NAME=cover-service-faktor-export
+COMPOSE_DOMAIN=cover-service-faktor-export.local.itkdev.dk
+
+# In all environments, the following files are loaded if they exist,
+# the latter taking precedence over the former:
+#
+#  * .env                contains default values for the environment variables needed by the app
+#  * .env.local          uncommitted file with local overrides
+#  * .env.$APP_ENV       committed environment-specific defaults
+#  * .env.$APP_ENV.local uncommitted environment-specific overrides
+#
+# Real environment variables win over .env files.
+#
+# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+#
+# Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
+# https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=53dd02f2c585e2d21b2d4fa1b8b5407a
+#TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
+#TRUSTED_HOSTS='^localhost|example\.com$'
+###< symfony/framework-bundle ###
+
+###> doctrine/mongodb-odm-bundle ###
+MONGODB_URL=mongodb://db:password@mongo:27017
+MONGODB_DB=db
+###< doctrine/mongodb-odm-bundle ###
+
+###> nelmio/cors-bundle ###
+CORS_ALLOW_ORIGIN='^https?://localhost(:[0-9]+)?$'
+###< nelmio/cors-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# For a PostgreSQL database, use: "postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8"
+# IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
+DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7
```

Comment:
Normally "we/I" would set the defaults here that matches the docker-compose setup for database connection.

Reply:
If doctrine (normal database url) is not used by the site it should be possible to remove this completely from the codebase.

Reply:
Removed doctrine orm integration. But I still use Doctrine with regards to mongodb